tracks/ruby/exercises/resistor-colors/mentoring.md

```diff
@@ -0,0 +1,33 @@
+Resistor Colors is a side exercise, unlocked by Hello World
+
+### New Concepts
+Array, Array#index, `map`, Constant, chaining methods
+
+### Minimal solution for approval
+```ruby
+
+class ResistorColors
+
+  COLOR_CODES = ["black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white"]
+  
+  def self.value(colors)
+    colors.map { |color| COLOR_CODES.index(color) }.join.to_i
+  end
+
+end
+
+```
+
```

Comment:
Add note about acceptable variations include using a `module`, `class << self`, `module_function` and the likes. We don't want mentors to comment on any of that, seeing it's such an early exercise!